src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RangeExtensions.cs

```diff
@@ -10,6 +10,12 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer
 {
     internal static class RangeExtensions
     {
+        public static readonly Range UndefinedRange = new Range
```

Comment:
Odd to put this in an extension class but I can't think of a better alternative.

Reply:
I wonder if this is something O# would be interested in including with the mainline O# release.